server/utils/emulator.go

```diff
@@ -271,3 +273,34 @@ func (m EmulatorAPIServer) ResetCodeCoverage(w http.ResponseWriter, r *http.Requ
 	m.emulator.ResetCoverageReport()
 	w.WriteHeader(http.StatusOK)
 }
+
+func (m EmulatorAPIServer) Logs(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	vars := mux.Vars(r)
+	id := vars["id"]
```

Comment:
Maybe check if the ID was provided and return a (ideally helpful) error (`StatusBadRequest`) if it is missing

Reply:
fixed at identifier check